resources/views/beheer/user/create_edit.blade.php

```diff
@@ -99,10 +99,6 @@
                         </div>
                     </div>
                 </div>
-                <div class="form-group col-md-4">
-                    {!! Form::label('gender', trans('user.gender')) !!}
-                    {!! Form::select('gender',["man" => trans('user.man'), "female" => trans('user.female')], ($user != null) ? $user->gender : "", ['class' => 'form-control','required' => 'required','id' => 'gender']) !!}
```

Comment:
Hoe ziet het formulier er uit nu dit weg is? Moet je dan de andere kolommen niet wat breder maken?

Reply:
Yes heb bij de pagina en backoffice pagina het naar md-6 gezet